quupe-RN/android/app/build.gradle

```diff
@@ -137,7 +137,12 @@ android {
 }
 
 dependencies {
+<<<<<<< HEAD
```

Comment:
This should not be here. Please solve your merge conflicts properly.

Reply:
I can not merge your pull request to develop branch because these lines are on GitHub. Please remove them. Since they are coming from your git commit, you can solve this. Let me know if you need help.